### DISCUSSION TITLE
Filter Commits by Filename

### DISCUSSION THREAD

**Discussion by 22decembre** [Xerdi/gitinfo-lua#6] [Q&A] [answered]
I wish to use that package in a project where there are several tex files, each representing one pdf in the end.

Can I make it so that, when I compile a tex file, only tags/commits related to that tex file appear in the listing ?

**Comment by MacLotsen** [comment, marked as answer]
Unfortunately not. That would require some kind of filtering feature, ~~which is not part of the scope~~.

It's now fixed in version 1.1.0. Thanks for your input!
